condorpy/htcondor_object_base.py

```diff
@@ -53,13 +54,43 @@ def cluster_id(self):
         """
         return self._cluster_id
 
+    @property
+    def num_jobs(self):
```

Comment:
New property for getting number of jobs (see implementations on Job and Workflow).